internal/blueprint/customizations.go

```diff
@@ -380,3 +380,10 @@ func (c *Customizations) GetOpenSCAP() *OpenSCAPCustomization {
 	}
 	return c.OpenSCAP
 }
+
+func (f *FDOCustomization) IsFDO() *FDOCustomization {
```

Comment:
I'd call this `HasFDO`

Reply:
make sense to me too, changed in the latest commit